client/src/components/formHelpers/form-validators.tsx

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 import { Trans } from 'react-i18next';
 
-// Matches editor links for: Replit, Glitch, CodeSandbox, GitHub
+// Matches editor links for: Replit, Glitch, CodeSandbox, GitHub. NOT Codespaces, and NOT Gitpod yet
+// Once safari allows negative lookbehinds, this can be used:
+// |(?<!https:\/\/\d+-[\w.-]+)\.gitpod\.io
```

Comment:
```suggestion
```
Unfortunately not, because https://github.com/3000 exists. Unless we know the structure of Gitpod's links, we can't reliably tell the difference between 3000's editor url and app url.